src/sas/example_data/media/testdata_help.rst

```diff
@@ -271,6 +300,38 @@ ISIS_98940
 
 .. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
 
+Magnetic Data
+^^^^^^^^^^^^^
+Polarised SANS data.
+
+Magnetic Test Data
+..................
+**Example data files are located in the \\magnetic_data sub-folder.**
+
+S50
+  - 2D and 1D sector data from 10 nm nanospheres in d8-toluene (c_Fe2O3 = 50 mg/ml,
+    ~1 vol%) at an aligning magnetic field of 1.5 T in horizontal direction.
+  - See `10.1088/1367-2630/14/1/013025 <https://dx.doi.org/10.1088/1367-2630/14/1/013025>`_
+
+.. ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+
+NR Data
+^^^^^^^
+Neutron (NR) or X-ray (XR) Reflectometry data sets primarily contain the
+interfacial reflectivity as a function of *Q*.
+
+.. note:: The Refl1D reflectivity model-fitting software uses the same fitting
+         engine (Bumps) as SasView.
+
+NR Test Data
```

Comment:
Why are needed both NR and NR Test Data? Can they not be put in the same folder?

Reply:
It's not that they are in different folders, rather that the original author of this help doc chose to use a type of data heading and then a test data subheading.